wrappers/winuser.go

```diff
@@ -552,3 +553,8 @@ func SwitchDesktop(desktop syscall.Handle) error {
 	}
 	return nil
 }
+
+func GetShellWindow() syscall.Handle {
+	r1, _, _ := syscall.Syscall(procGetShellWindow.Addr(), 0, 0, 0, 0)
+	return syscall.Handle(r1)
+}
```

Comment:
Let's put this in alphabetical order.  Also, there's a missing newline at EOF.

Reply:
Why in alphabetical order? I thought it should be in the same order as in the file WinUser.h ...

Reply:
It's a standard that we use throughout the gowin32 library.

Reply:
Hmm but other functions in this file are not in alphabetical order but in order like in WinUser.h. Why ?

Reply:
It looks like the functions in this file started out alphabetized and then stuff got added out of order.  If you look at other files like winbase.go or winsvc.go, the functions are all alphabetized.  It's presumably my mistake for not catching it on prior code reviews.  Let's go ahead and merge this as is, and I can clean up winuser.go as a separate commit.

Reply:
I'm looking to  winbase.go and it also has wrong order (look on SetProcessShutdownParameters or AllocateAndInitializeSid, CheckTokenMembership and other)
I remember you wrote that the order should be the same as in the .h files, which was quite tiring to check ... To sum up: should we definitely stick to the alphabetical order?

Reply:
That's because there's two different libraries that have functions in winbase.go.  First come the functions from kernel32.dll in alphabetical order, followed by the functions from advapi32.dll in alphabetical order.  Perhaps I should add some comments to clarify this.

If I said something about matching the ordering from Microsoft's header files, that was probably in reference to structures and constants.  Since these can have dependencies among them, alphabetizing them doesn't work, but it's fine for functions.